migrations/0004_remove_mydec_flags.py

```diff
@@ -0,0 +1,80 @@
+# TODO: Add context
+import os
+
+import awswrangler as wr
+from pyathena import connect
+from pyathena.pandas.cursor import PandasCursor
+
+pedantic_matt_path = os.path.join(
+    os.getenv("AWS_BUCKET_SV_BACKUP"),
+    "0004_remove_mydec_flags",
+    "old_data_file",
+    "2024-12-10_14:17-pedantic-matt.parquet",
+)
+
+compassionate_rina_path = os.path.join(
+    os.getenv("AWS_BUCKET_SV_BACKUP"),
+    "0004_remove_mydec_flags",
+    "old_data_file",
+    "2024-12-10_13:30-compassionate-rina.parquet",
+)
+
+df_to_edit_pedantic_matt = wr.s3.read_parquet(pedantic_matt_path)
+df_to_edit_compassionate_rina = wr.s3.read_parquet(compassionate_rina_path)
+
+# # # - - - - - -
+# The task here is to, after flagging 2023 and onward ias sales,
+# find flagged doc_no's that are in both of the above files
+# and NOT in default.vw_pin_sale, and then remove those observations
+#
+# These sales in theory should be isolated to 2023 and 2024
+#
+# # # - - - - - -
```

Comment:
When you query `select * from sale.metadata where run_id = '2024-12-10_13:30-compassionate-rina.parquet'`, you'll see a `run_note` that links to [here](https://github.com/ccao-data/model-sales-val/pull/135). 

Should we document somewhere that we made this migration and eliminated these flags? Or is the `run_note` description sufficient? I could leave a comment on the link or I could manually edit the `run_note`